src/app/Exceptions/Handler.php

```diff
@@ -32,10 +33,55 @@ class Handler extends ExceptionHandler
      *
      * @return void
      */
-    public function register()
+    public function register(): void
+    {
+    }
+
+    /**
+     * report
+     */
+    public function report(Throwable $e): void
     {
-        $this->reportable(function (Throwable $e) {
-            //
-        });
+        parent::report($e);
+    }
+
+    /**
+     * render
+     */
+    public function render($request, Throwable $e): Response
+    {
+        return parent::render($request, $e);
+    }
+
+    /**
+     * HTTP以外の例外発生時の処理
+     */
+    protected function prepareResponse($request, Throwable $e): Response
+    {
+        // HTTPでない例外が起これば、HTTP例外の500に変更する
+        if (!$this->isHttpException($e) && !config('app.debug')) {
+            $e = new HttpException(500, $e->getMessage(), $e);
```

Comment:
`500`はマジックナンバーだから、下記の定数で表現すると良いかも！
```suggestion
use Illuminate\Http\Response; // ファイル上部のuse側で定義

            $e = new HttpException(Response::HTTP_INTERNAL_SERVER_ERROR, $e->getMessage(), $e);
```
https://github.com/symfony/symfony/blob/6.2/src/Symfony/Component/HttpFoundation/Response.php#L24-L86